CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the LaunchDarkly git-flag-parser will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org).
 
+## [0.1.1] - 2019-01-10
+### Changed
+- `updateSequenceId` is now an optional parameter. If not provided (or set to a number < 0), data will always be updated. If provided, data will only be updated if the existing `updateSequenceId` is less than the new `updateSequenceId`.
```

Comment:
whats the rationale for making negative numbers always trigger updates?

Reply:
This is sort of a workaround to account for command-line options requiring a default value of the underlying type. Since we can't give pointer values to the `flag` package, using negative numbers to indicate that there is no update sequence id. Another alternative here would be to just have the underlying type of the option be a string, but that would affect the auto-generated "help" documentation of the command line tool

Reply:
you can't specify that the command-line option is optional? if its optional wouldn't it require a default value?